src/filters.js

```diff
@@ -398,10 +398,21 @@ var filters = {
         return r.copySafeness(obj, obj);
     },
 
-    striptags: function(input) {
+    striptags: function(input, preserve_linebreaks) {
         input = normalize(input, '');
+        preserve_linebreaks = preserve_linebreaks || false;
         var tags = /<\/?([a-z][a-z0-9]*)\b[^>]*>|<!--[\s\S]*?-->/gi;
-        return r.copySafeness(input, filters.trim(input.replace(tags, '')).replace(/\s+/gi, ' '));
+        var res = '';
+        if (preserve_linebreaks) {
+            res = filters.trim(input.replace(tags, ''))
```

Comment:
Why not extract the common `filters.trim(input.replace(tags, ''))` outside of the if/else?